src/ibusenginesimple.c

```diff
@@ -471,6 +495,11 @@ ibus_engine_simple_update_preedit_text (IBusEngineSimple *simple)
                             g_string_append_c (s, ' ');
                         g_string_append_unichar (s, ch);
                     }
+                } else if (keysym >= 0x0100fdd0 && keysym <= 0x0100fdd9) {
+                    /* A BEPOs pseudo deadkey */
```

Comment:
Again, why not use a more generic code? E.g. use Unicode char category to detect Unicode non-characters.